src/pages/ProductList/components/SubLnb/SubLnb.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import './SubLnb.scss';
+const SubLnb = () => {
+  return (
+    <div className="subLnb">
+      <ul className="forPc">
+        <li className="title">제품</li>
+        <li className="handlersThirdMenuLink">인센스</li>
+        <li className="handlersThirdMenuLink">콘</li>
+        <li className="handlersThirdMenuLink">홀더</li>
+      </ul>
```

Comment:
```suggestion
      <ul className="title">
        제품
        <li className="handlersThirdMenuLink">인센스</li>
        <li className="handlersThirdMenuLink">콘</li>
        <li className="handlersThirdMenuLink">홀더</li>
      </ul>
```
이렇게 되야 제품은 리스트의 title이 되고, 
인센스,콘,홀더가 하위 리스트가 되는것 같은데 어떠세요?